src/main/groovy/com/bmuschko/gradle/clover/InstrumentCodeAction.groovy

```diff
@@ -95,7 +95,13 @@ class InstrumentCodeAction implements Action<Task> {
                 }
 
                 getMethodContexts().each {
-                    ant.methodContext(name: it.name, regexp: it.regexp)
+                    ant.methodContext(
```

Comment:
Cool. I'll update the README to reflect this dependency. Thanks.
On Sun, Jan 18, 2015 at 8:11 AM Benjamin Muschko <notifications@github.com>
wrote:

> In src/main/groovy/com/bmuschko/gradle/clover/InstrumentCodeAction.groovy
> <https://github.com/bmuschko/gradle-clover-plugin/pull/56#discussion-diff-23133178>
> :
>
> > @@ -95,7 +95,13 @@ class InstrumentCodeAction implements Action<Task> {
> >                  }
> >
> >                  getMethodContexts().each {
> > -                    ant.methodContext(name: it.name, regexp: it.regexp)
> > +                    ant.methodContext(
>
> No, we don't need to support older versions. That just makes the code more
> complex. IMHO there's no good reason for staying on an older version of
> Clover.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bmuschko/gradle-clover-plugin/pull/56/files#r23133178>
> .
>